### DISCUSSION TITLE
GrapesJS (TypeScript)

### DISCUSSION THREAD

**Discussion by MuhammadIsmail-Dev** [GrapesJS/grapesjs#4353] [Q&A] [answered]
### Need Help
hi great people.!

- [x] using latest version

I am using grapesJS in my project with **typescript**. 
I am facing a problem related to "Types".
**two of them:**
> file index.d.ts is not a module.
> Type '[string]' is not assignable to type '[]'.

### Cause:
these typings in index.d.ts are fixed here in the repository but aren't updated in **npm**.

**PLEASE UPDATE GRAPESJS ON NPM** so I can update it from npm and get rid of the problem and can continue my work on grapesJS 

thanks

**Comment by artf** [comment, marked as answer]
Yeah it will be fixed in the next release

**Reply by MuhammadIsmail-Dev** [reply]
Sounds good I hope it'll be released soon thus I can install it from NPM instead of copying file from here (repo)
